src/ui/screens/DemocracyScreen.test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {NavigationProp} from '@react-navigation/native';
+import {DashboardStackParamList} from '@ui/navigation/navigation';
+import {fireEvent, render} from 'src/testUtils';
+import {DemocracyScreen} from './DemocracyScreen';
+import {ReactTestInstance} from 'react-test-renderer';
+import {Linking} from 'react-native';
+
+jest.useFakeTimers();
```

Comment:
Why do you need this?

Reply:
We need this, or else we get a warning of imports being torn down. Since these components have the data when the component is rendered.